chart/templates/rabbitmq-autoscaling.yaml

```diff
@@ -0,0 +1,45 @@
+{{- if .Values.rabbitmqAutoscaling.enabled }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: keda-rabbitmq-secret
+data:
+  host: "{{ $.Values.commonEnv.CLUSTER_WIDE_BROKER_URL | b64enc }}"
+---
+apiVersion: keda.sh/v1alpha1
+kind: TriggerAuthentication
+metadata:
+  name: keda-trigger-auth-rabbitmq-conn-new
+  namespace: default
+spec:
+  secretTargetRef:
```

Comment:
Can you use `env` here? The separate `BROKER_URL` and `CLUSTER_WIDE_BROKER_URL` doesn't seem right + updating secrets is always a pain https://keda.sh/docs/2.11/concepts/authentication/#environment-variables

Reply:
done!